laze-project.yml

```diff
@@ -12,8 +12,8 @@ contexts:
       - ?defmt
     env:
       RUSTFLAGS:
-        - "-Clink-arg=--nmagic"
-        - "-Clink-arg=${LINK_ARG_PREFIX}--no-eh-frame-hdr"
+        #        - "-Clink-arg=--nmagic"
+        #        - "-Clink-arg=${LINK_ARG_PREFIX}--no-eh-frame-hdr"
```

Comment:
@kaspar030 I copied this from your WIP branch for xtensa support.
I am not familiar enough with linking in general to really understand if/ when these flags are needed.

Reply:
I have now moved `-Clink-arg=--nmagic` to the `cortex-m` module because [according to the rustc book](https://doc.rust-lang.org/beta/rustc/platform-support/arm-none-eabi.html?search=eh-frame-hdr) it's part of the usual **Arm** bare-metal linker setup.
I looked into when/ why the `--no-eh-frame-hdr` flag is needed, and seems like it's only helpful for reducing code size, but I didn't find any docs that actually recommend using it. Might be just a premature optimization. Can anyone give some input/ opinion on whether we have to keep it?

Reply:
Let's keep it for now and investigate another time. Moving to "cortex-m" is fine here.